app/src/main/java/woowacourse/shopping/data/repository/OrderRepository.kt

```
@@ -1,35 +1,20 @@
-                    call: Call<Unit>,
-                    t: Throwable,
-                ) {
-                    callback(Result.failure(t))
-                }
-            },
-        )
-    }
+            if (response.isSuccessful) {
+                Unit
+            } else {
+                throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)